composer.lock

```diff
@@ -1557,7 +1557,7 @@
         },
         {
             "name": "symfony/asset",
-            "version": "v4.2.4",
+            "version": "v4.2.7",
```

Comment:
Nėra problema, bet užduotis `composer update` nereikalavo.

Yra gera praktika: daryti mažomis dalimis ir pabaigus pasižiūrėti, ar pakeista tik tai, ko reikia – nes po to irgi kiti skaitys tavo kodą.

Plačiau:
 * https://help.github.com/en/articles/about-pull-request-reviews
 * https://en.wikipedia.org/wiki/KISS_principle